lib/css/components/links.less

```diff
@@ -1,214 +1,102 @@
-    -moz-appearance: none;
-    background: transparent;
-    border: 0;
-    padding: 0;
-    line-height: inherit;
-    user-select: auto;
-    font-family: inherit;
-
-    &:focus {
-        outline: none;
-    }
-}
-
-.s-anchors {
-    .colorize-all(@name, @normal, @hover) {
-        &.s-anchors__@{name},
-        & .s-anchors.s-anchors__@{name} {
-            a:not(.s-link),
-            .s-btn.s-btn__link {
-                color: @normal;
 
-                &:hover,
-                &:active {
-                    color: @hover;
-                }
-
-                &:visited {
-                    color: @normal;
-
-                    &:hover,
-                    &:active {
-                        color: @hover;
-                    }
-                }
-            }
-        }
+    &:visited {
+        color: var(--_li-fc-visited);
     }
-    &.s-anchors__underlined {
-        a:not(.s-link),
-        .s-btn.s-btn__link {
-            text-decoration: underline;
-        }
-    }
-    .colorize-all(default, var(--theme-link-color), var(--theme-link-color-hover));
-    .colorize-all(grayscale, var(--black-700), var(--black-600));
-    .colorize-all(inherit, inherit, inherit);
-    .colorize-all(muted, var(--black-500), var(--black-400));
-    .colorize-all(danger, var(--red-500), var(--red-400));
-}
 
-.s-block-link {
-    display: block;
-    width: 100%;
-    color: var(--black-600);
-    padding: var(--su6) var(--su12);
-
-    // Normalize for when applying to a button
+    color: var(--_li-fc);
     cursor: pointer;
-    border: none;
-    background: transparent;
-    border-radius: 0;
-    text-align: left;
-    line-height: inherit;
-    font-family: inherit;
-
-    &:hover,
-    &:active {
-        color: var(--black-800);
-    }
-
-    @focus-styles();
-
-    &.is-selected {
-        --_li-block-bs: inset var(--_li-block-bs-offset-x, 3px) 0 0 var(--theme-primary-color);
-
-        color: var(--black-800);
-        font-weight: bold;
-        background-color: var(--black-050);
-
-        .dark-mode({
-            background-color: var(--black-025);
-        });
-
-        &.s-block-link__right {
-            --_li-block-bs-offset-x: -3px;
-        }
-
-        &.s-block-link__left,
-        &.s-block-link__right {
-            box-shadow: var(--_li-block-bs);
-
-            &:focus:not(:focus-visible),
-            &:focus-visible {
-                outline: none;
-            }
+    text-decoration: none;
+    user-select: auto;
+};
 
-            &:focus-visible {
-                box-shadow: var(--_li-block-bs), 0 0 0 var(--su-static4) var(--focus-ring-muted);
-            }
+.s-link {
+    button& {
+        &:focus {
+            outline: none;
         }
-    }
-
-    &.s-block-link__danger {
-        color: var(--red-500);
 
-        &:hover,
-        &:active {
-            color: var(--red-700);
-        }
+        appearance: none; // TODO in the PR, explain the removal of `-moz` and `-webkit` prefixes here
```

Comment:
We previously used vendor prefixes here. That's no longer necessary, since we only support browsers that play nice with plain old `appearance`.